docs/spl-docs/time-modifiers.md

```diff
@@ -0,0 +1,177 @@
+### Time modifiers and the Time Range Picker
+
+The time range that is specified in the query using time modifiers, or in a saved search, overrides the time range that is selected in the Time Range Picker.
+
+For example, if you specify a time range of Last 24 hours in the Time Range Picker and in the Search bar you specify `earliest=-1h latest=now`, the search only looks at events that have a timestamp within the last 1 hour.
+
+
+## Syntax
+
+### Specify absolute time ranges
+
+For exact time ranges, the syntax for the time modifiers is `%m/%d/%Y:%H:%M:%S`. For instance, to search within a time frame from midnight on May 1, 2023, to midnight on May 10, 2023, you would use the following syntax:
+
+```spl
+earliest=05/01/2023:00:00:00 latest=05/10/2023:00:00:00
+```
+
+If you specify only the earliest time modifier, latest is set to the current time `now` by default. If you specify a latest time modifier, you must also specify an earliest time. If the specified time modifiers are invalid, then the system would return an error.
+
+### Specify relative time ranges
+
+The syntax for defining relative time ranges is an integer and a time unit.
+
+1. Begin your relative time modifier String with a minus (-) or a plus (+) sign to indicate the offset before (-) or after (+) the time amount.
+
+2. Specify the time amount by combining a number with a time unit. When single time units are used, the number 1 is implied. For instance, `s` is equivalent to `1s`, `m` to `1m`, and so on.
+
+The supported time units are listed in the following table: 
+
+| Time range | Valid values                                  |
+|------------|-----------------------------------------------|
+| seconds    | `s`, `sec`, `secs`, `second`, `seconds`       |
+| minutes    | `m`, `min`, `minute`, `minutes`               |
+| hours      | `h`, `hr`, `hrs`, `hour`, `hours`             |
+| days       | `d`, `day`, `days`                            |
+| weeks      | `w`, `week`, `weeks`                          |
+| months     | `mon`, `month`, `months`                      |
+| quarters   | `q`, `qtr`, `qtrs`, `quarter`, `quarters`     |
+| years      | `y`, `yr`, `yrs`, `year`, `years`             |
+
+When specifying relative time, use `now` to refer to the current time.
+
+### Relative time modifiers that snap to a time
+
+In relative time settings, you can specify a snap to time, which acts as an offset from the relative time. This snap to time unit will round down to the nearest or latest time for the time amount that you specify. To apply this, simply separate the time amount from the snap to time unit using an `@` symbol.
+
+The syntax for the snap to time unit is `[+|-]<time_integer><time_unit>@<time_unit>`.
+
+When snapping to the nearest or latest time, the system always snaps backwards or rounds down to the latest time that is not after the specified time. For example, the current time is 15:45:00 and the snap to time is earliest=-h@h. The time modifier snaps to 14:00.
```

Comment:
nit: use the same format for `15:45:00` and `14:00` (either include seconds in both, or in neither)